book/blocks.md

```diff
@@ -1101,7 +1101,7 @@ From
 ```php
 protected function blockAccess(AccountInterface $account) {
   /** @var \Drupal\node\Entity\Node $node */
-  $node = \Drupal::routeMatch()->getParameter('node');
+  $node = $this->getContext('node');
   if ($node) {
```

Comment:
This code is taken from the RSVP Module, specifically this block:
https://git.drupalcode.org/project/rsvp_module/-/blob/1.0.x/src/Plugin/Block/RSVPBlock.php

I understand what you are saying here, but I think it would be best to add the section on the context_definitions before this block of code to properly show the relationship between them.

It's a good tip though. Might also be worth contributing back to the RSVP module :)